src/app/auth/PageLogin.tsx

```diff
@@ -21,6 +23,26 @@ export const PageLogin = () => {
         <Box p="6" borderRadius="md">
           <LoginForm onSuccess={onLogin} />
         </Box>
+        <Box>
+        Discover {' '}
+          <Link
+            color="violet"
+            href={bearstudioLink}
+            isExternal
+          >
+            who we are !
+          </Link>
+        </Box>
+        <Box>
+        Look at this project {' '}
+          <Link
+            color="violet"
+            href={repositoryLink}
+            isExternal
+          >
+            gitHub repository !
+          </Link>
+        </Box>
```

Comment:
pour le design je ne suis pas d'accord .
* pour le lien github de repo , la plus part de temps , c'est une icon de github au lieu de texte
* pour le text discover discover who we are , peut être un texte + le logo de bs

Reply:
Pour moi pas besoin du logo, par contre c'est mieux de mettre le nom de l'entreprise.

Niveau design, il faudrait que ce soit en mode footer et pas juste en dessous du bloc, mais vraiment en bas de la page.

Proposition de wording:

![image](https://github.com/user-attachments/assets/1aa891b0-d910-4b8b-a35b-48faee6082e9)

Reply:
Est-ce que je change le lien vers le site bearstudio ? (au lieu d'afficher la team, afficher directement la page d'accueil du site)

Reply:
oui ,ça serait mieux